### PR TITLE
fix/AACT-228 restrict access admin

### PR DESCRIPTION
**Pull request by Javier-Jimenez-18**
Added an admin column to users. 
Removed is_admin? before_action filter from Application Controller. 
Removed version number from migrations because it was causing errors.

### CODE REVIEW

**Review by micronix — CHANGES_REQUESTED** [review]
Test the app with an admin user and a non-admin user

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::Base
+  before_action :is_admin?
```
[the remark this answers is not in the document]
Comment (Javier-Jimenez-18) [reply app/controllers/application_controller.rb]: Removed is_admin? from application controller.
Tested the app with an admin user and non-admin user.

**Comment by Javier-Jimenez-18** [comment]
Tested the app with a user not logged in and a user logged in.
Tested the app with an admin user and a non-admin user.

Screen Shot 2021-12-16 at 11 42 12 AM
Screen Shot 2021-12-16 at 11 43 37 AM
Screen Shot 2021-12-17 at 8 31 29 AM

**Review by crystal-wb — APPROVED** [review]
Looks good. Nice work

**Pull request merged**